src/app/theme/components/baMenu/components/baMenuItem/baMenuItem.html

```diff
@@ -1,15 +1,15 @@
 <li *ngIf="!menuItem.hidden" [title]="menuItem.title" [ngClass]="{'al-sidebar-list-item': !child, 'ba-sidebar-sublist-item': child, 'selected': menuItem.selected && !menuItem.expanded, 'with-sub-menu': menuItem.children, 'ba-sidebar-item-expanded': menuItem.expanded}">
```

Comment:
Should be `[title]="menuItem.title | translate"` to display correct text when hover on menu item.
Btw, Thanks for great work!